contracts/wormhole/contracts/mapping-token/v2/Backing.sol

```diff
@@ -8,57 +8,30 @@ import "../interfaces/ICrossChainFilter.sol";
 import "../interfaces/IFeeMarket.sol";
 import "../interfaces/IMessageCommitment.sol";
 import "../interfaces/IOutboundLane.sol";
-import "../../utils/DailyLimit.sol";
 import "../../utils/Ownable.sol";
 import "../../utils/Pausable.sol";
-import "../interfaces/IBacking.sol";
-import "../interfaces/IERC20.sol";
-import "../interfaces/IGuard.sol";
 import "../interfaces/IInboundLane.sol";
-import "../interfaces/IMappingTokenFactory.sol";
 
-contract Backing is Initializable, Ownable, DailyLimit, ICrossChainFilter, IBacking, Pausable {
+contract Backing is Initializable, Ownable, ICrossChainFilter, Pausable {
```

Comment:
This implementation looks very generalize and it can be merged with MappingTokenFactory.

We should call them ``CrosschainApp`,  `HelixApp` or something else.

Reply:
They are merged together in `HelixApp.sol`.
Now we can see both Backing and MappingTokenFactory are HelixApp instance.
And they have the same identity verification from each other.